scripts/linux/makefile.pynq

```diff
@@ -163,13 +162,12 @@ doc_notebooks:
 	${FINAL_NOTEBOOKS_DIR}/getting_started/2_programming_python.ipynb
 	cp ${REPO_DIR}/docs/source/5_programming_onboard.ipynb \
 	${FINAL_NOTEBOOKS_DIR}/getting_started/3_programming_onboard.ipynb
-	cp ${REPO_DIR}/docs/source/6a_base_overlay_iop.ipynb \
```

Comment:
I see these are the source file names in the image_2017_01 branch ... let me know if breaking update flow.